CHANGELOG.md

```diff
@@ -14,6 +15,106 @@
+- Types `ConditionRouteAccepted` and `ConditionRouteResolvedRefs` are now
+  deprecated in favor of `RouteConditionAccepted` & `RouteConditionResolvedRefs`
+  [#1114](https://github.com/kubernetes-sigs/gateway-api/pull/1114)
+- The Gateway API webhook is now deployed in a `gateway-system` namespace
+  instead of `gateway-api`.
+  [#1051](https://github.com/kubernetes-sigs/gateway-api/pull/1051)
+- the `ParentRef` type was renamed to `ParentReference`
+  [#982](https://github.com/kubernetes-sigs/gateway-api/pull/982)
+- Gateway API now includes "Experimental" release channel
+  [#945](https://github.com/kubernetes-sigs/gateway-api/pull/945)
+- Gateway API CRDs now include `gateway.networking.k8s.io/bundle-version` and
+  `gateway.networking.k8s.io/channel` annotations
+  [#945](https://github.com/kubernetes-sigs/gateway-api/pull/945)
+- URL Rewrites and Path redirects have been added as new "Experimental" features
+  [#945](https://github.com/kubernetes-sigs/gateway-api/pull/945)
+- The v1alpha1 API version has been deprecated.
+  [#906](https://github.com/kubernetes-sigs/gateway-api/issues/906)
+
+### Improvements
+
+- Conformance tests were introduced with [GEP-917][gep-917] and multiple
+  conformance tests were added from a variety of contributors under the
+  `conformance/` directory.
+- The status of the GatewayClass "Accepted" condition for the `GatewayClass`
+  is now present in `kubectl get` output.
+  [#1168](https://github.com/kubernetes-sigs/gateway-api/pull/1168)
+- New `RouteConditionReason` types `RouteReasonNotAllowedByListeners` and
+  `RouteReasonNoMatchingListenerHostname` were added.
+  [#1155](https://github.com/kubernetes-sigs/gateway-api/pull/1155)
+- New `RouteConditionReason` type added with `RouteReasonAccepted`,
+  `RouteReasonResolvedRefs` and `RouteReasonRefNotPermitted` constants.
+  [#1114](https://github.com/kubernetes-sigs/gateway-api/pull/1114)
+- Routes can now select `Gateway` listeners by port number
+  [#1002](https://github.com/kubernetes-sigs/gateway-api/pull/1002)
+- Introduced PreciseHostname which prevents wildcard characters in relevant
+  Hostname values.
+  [#956](https://github.com/kubernetes-sigs/gateway-api/pull/956)
+
+[gep-917]:https://gateway-api.sigs.k8s.io/geps/gep-917/
+
+### Bug Fixes
+
+- Fixes a problem that would cause webhook deployment to fail on Kubernetes
+  v1.22 and greater.
+  [#991](https://github.com/kubernetes-sigs/gateway-api/pull/991)
+- Fixes a bug where the `Namespace` could be unspecified in `ReferencePolicy`
+  [#964](https://github.com/kubernetes-sigs/gateway-api/pull/964)
+- Fixes a bug where v1alpha2 GatewayClass controller names were not being
+  shown in the output of `kubectl get gatewayclasses`
+  [#909](https://github.com/kubernetes-sigs/gateway-api/pull/909)
+
+### Documentation
+
+- Blog post added for the new `v1alpha2` API
+  [#904](https://github.com/kubernetes-sigs/gateway-api/pull/904)
+- Various minor corrections to documentation for visual issues and typos
```

Comment:
I think I'd leave doc changes out of changelog.

Reply:
```suggestion
### Documentation

- Blog post added for the new `v1alpha2` API
  [#904](https://github.com/kubernetes-sigs/gateway-api/pull/904)
```